src/components/cart/CartItemsList.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import CartItem from "./CartItem";
+
+function CartItemsList(props){
+
+    const { cartData, handleQuantityUpdate } = props;
+    
+    const cartItems = cartData.map( item => {
```

Comment:
may be you can keep it in a fn to get the jsx (usually I keep it in the same file, but it's again a discussion topic)

Reply:
both would have same effect i think